components/HourCounter.js

```diff
@@ -0,0 +1,17 @@
+let hours = 0;
+const counter = document.createElement("h2");
+
+counter.id = "counter";
+
+const updateCounter = (activities) => {
+  hours = activities.reduce((acc, obj) => {
+    return (acc += parseInt(obj.tempo));
+  }, 0);
```

Comment:
:nail_care: 
Voce pode dar uma resumida no seu código e tambem deixar as variaveis do reduce um pouco mais "legíveis"

```suggestion
  hours = activities.reduce(
    (total, activity) => total += parseInt(activity.tempo)
  , 0);
```